### ISSUE TITLE
no make_entity_deps and no utils

### ISSUE THREAD

**Issue by asharkinasuit** [amir-zeldes/xrenner#65]
In the [documentation](https://corpling.uis.georgetown.edu/xrenner/doc/models.html#entity-deps-tab), it says there is a script that will make the entity_deps.tab that can be found in utils. I've searched the repo and neither the master branch nor the develop branch seems to have either a util directory or the script mentioned.

Incidentally, it seems the Universal Dependencies have been updated since xrenner was written to use (slightly) different function labels, [see](http://universaldependencies.org/u/dep/all.html). Dependency trees I've generated with a different tool use this new format, where in particular all the prepositions seem to be marked simply as `case`...

**Comment by amir-zeldes** [comment]
Hi @asharkinasuit , thanks for the feedback. You're right, there is an unreleased folder of utilities I've been meaning to publish but haven't gotten around to. I added the script you're looking for here: 4bc408a8b83b1eafe26c0b188a7282ebf23fa470 . Some more useful utilities should follow in the next few weeks.

As for the dependencies for English, the default model expects Basic Stanford Typed Dependencies, not Universal Dependencies. I just added that to the README in the develop branch, I'll also add a note to the online documentation. If you modify the configuration for universal dependencies, I'm happy to add it as an alternative model in an upcoming release (in fact, the supplied Coptic mode uses UD and should be similar).
